docs/development.md

```diff
@@ -4,21 +4,47 @@ This document describes how to develop and add features to the Bank of Sirius ap
 
 ## Prerequisites 
 
-1. [A GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#console), connected to your billing account. 
-2. [A GKE cluster in your project](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster#gcloud). We recommend 4 nodes, machine type: `e2-standard-4`. 
+1. A clone of this repository.
+2. A linux build environment.
 
 ## Install Tools 
 
 You can use MacOS or Linux as your dev environment - all these languages and tools support both. 
 
-1. [Docker Desktop](https://www.docker.com/products/docker-desktop) 
-1. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (can be installed separately or via [gcloud](https://cloud.google.com/sdk/install)) 
-1. [skaffold **1.27+**](https://skaffold.dev/docs/install/) (latest version recommended)
-1. [JDK **14**](https://www.oracle.com/java/technologies/javase/jdk14-archive-downloads.html) (newer versions might cause issues)
-1. [Maven **3.6**](https://downloads.apache.org/maven/maven-3/) (newer versions might cause issues)
-1. [Python3](https://www.python.org/downloads/)  
-1. [piptools](https://pypi.org/project/pip-tools/)
-
+1. [Docker](https://www.docker.com/products/docker-desktop) 
+2. [GNU Make 4.x](https://www.gnu.org/software/make/)
+5. [JDK **14**](https://www.oracle.com/java/technologies/javase/jdk14-archive-downloads.html) (newer versions might cause issues)
```

Comment:
Right now we are using the JDK 8. As we benchmark, I thought we could upgrade to see how a new JDK affects perf.